modules/hosted-control-planes-overview.adoc

```diff
@@ -6,14 +6,11 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="hosted-control-planes-overview_{context}"]
-= Introduction to hosted control planes (Technology Preview)
+= Introduction to hosted control planes
 
 You can use hosted control planes for Red Hat {product-title} to reduce management costs, optimize cluster deployment time, and separate management and workload concerns so that you can focus on your applications.
 
-You can enable hosted control planes as a Technology Preview feature by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#cluster_mce_overview[multicluster engine for Kubernetes operator version 2.0 or later] on Amazon Web Services (AWS), bare metal by using the Agent provider, or {VirtProductName}.
-
-:FeatureName: Hosted control planes
-include::snippets/technology-preview.adoc[]
+You can enable hosted control planes by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#cluster_mce_overview[multicluster engine for Kubernetes operator version 2.0 or later] on bare metal by using the Agent provider, or {VirtProductName}. Hosted control planes is available as a Technology Preview feature on the Amazon Web Services and {ibm-z-title} platforms.
```

Comment:
"Amazon Web Services" -> "Amazon Web Services (AWS)" or attribute?

Reply:
Good catch. I didn't think to check for an attribute. It looks like there is one. I'll update that. Thanks!